### PR TITLE
configuration of tight coincidence

### PR DESCRIPTION
**Pull request by feigaodm**
@jhowl01 , please checkout this PR to study effect of tight coincidence on acceptance.

### CODE REVIEW

**Comment by feigaodm** [comment]
@jhowl01 I made the change as placeholder for later modifications, effect should be identical to default code, can you help to review?

File: pax/plugins/signal_processing/BuildPeaks.py [hunk 1]
```diff
@@ -31,9 +31,6 @@ def startup(self):
         self.first_top_ch = np.min(np.array(self.config['channels_top']))
         self.last_top_ch = np.max(np.array(self.config['channels_top']))
 
-        # tight coincidence
-        self.tight_coincidence_window = self.config.get('tight_coincidence_window', 50) // self.dt
```
Comment (jhowl01) [inline pax/plugins/signal_processing/BuildPeaks.py]: Why was this ever here? It seems it was never used?

File: pax/plugins/peak_processing/ClassifyPeaks.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def transform_event(self, event):
             # classification based on rise_time and aft
             if -peak.area_decile_from_midpoint[1] < self.s1_rise_time_bound(peak.area):
                 # Peak rises fast, could be S1
-                if peak.tight_coincidence <= 1:
+                if peak.tight_coincidence <= 2:
```
Comment (jhowl01) [inline pax/plugins/peak_processing/ClassifyPeaks.py]: Possible to move this number to the default config?

File: pax/plugins/signal_processing/BuildPeaks.py (hunk 1 above)
Comment (feigaodm) [reply pax/plugins/signal_processing/BuildPeaks.py]: I was trying to add this in the classification decision during clustering, but later found out it's not necessary. But forgot to cut it away.

**Pull request merged**